build/common.sh

```diff
@@ -189,8 +189,8 @@ function kube::build::verify_prereqs() {
 
 function kube::build::docker_available_on_osx() {
   if [[ -z "${DOCKER_HOST}" ]]; then
-    if [[ -S "/var/run/docker.sock" ]]; then
-      kube::log::status "Using Docker for MacOS"
+    if [[ -S "/var/run/docker.sock" ]] || [[ -S "$(docker context inspect | jq -r '.[0].Endpoints.docker.Host' | awk -F 'unix://' '{print $2}')" ]]; then
```

Comment:
How about using `--format` option?
`docker context inspect --format '{{.Endpoints.docker.Host}}'`
You can get the same information without jq command.

Reply:
great tip @t-inu 👍 I will update